src/common/components/select-repository-list/index.js

```diff
@@ -116,24 +116,19 @@ export class SelectRepositoryListComponent extends Component {
           <HeadingThree>
             { selectedRepositories.length } selected
           </HeadingThree>
-          <div className={ styles['footer-right'] }>
-            <div className={ styles['button-wrapper'] }>
-              { ids && ids.length > 0 &&
-                <LinkButton appearance="neutral" to={`/user/${user.login}`}>
-                  Cancel
-                </LinkButton>
-              }
-            </div>
-            <div className={ styles['button-wrapper'] }>
-              <Button
-                appearance={ 'positive' }
-                disabled={ !selectedRepositories.length || buttonSpinner }
-                onClick={ this.handleAddRepositories.bind(this) }
-                isSpinner={buttonSpinner}
-              >
-                Add
-              </Button>
-            </div>
+          <div className={ styles.buttons }>
+            <LinkButton appearance="neutral" to={`/user/${user.login}`}>
+              Cancel
+            </LinkButton>
+            {' '}
```

Comment:
Putting space between inline buttons (to avoid additional wrappers for spacing).